backend/django/create-user-model.mo.md

```diff
@@ -0,0 +1,74 @@
+# Context:
+You have a Django project installed. You need to handle different types of users
+
+# Prerequisite:
+Have the project installed.
+https://github.com/bamlab/dev-standards/blob/master/backend/django/getting-started.mo.md
+
+# Situation:
+You have a basic user model with a name.
+You want to create another type of user, a Mayor that has a City. 
+You do not want to override the classic user model.
+You want to create a Mayor from the admin
+You want the newly created Mayor to be part of the staff (User that can access the admin)
+
+## Add the Mayor model
+
+- In the models.py file, where the User is defined, create a new class that inherits the User:
+```
+class Mayor(User):
+    city = models.ForeignKey('locations.city')
+
+    class Meta:
+        verbose_name = "Mayor"
+```
+ 
+## Create a Mayor from the admin
+
+- In the admin.py, where the User model is added `admin.register(User)`
+- Add the following:
+```
+@admin.register(Mayor)
+class MyMayorAdmin(UserAdmin):
+    pass
+```
+
+You now see the Mayor in the admin. But all the forms (form to add a Mayor, form to change/update a Mayor) are the same as for the regular User.
+
+## Override the admin form to add a Mayor
+
+- In the admin.py when you declared your MyMayorAdmin class, you can choose the fields you want to display in the form by overriding the "add_fieldsets" property.
+- In our case:
+```
```

Comment:
same, python highlight

Reply:
And same for import. I would like the MO to be easily reusable and for that the reader should not search for the imports.